### PR TITLE
Add Marathon properties to DCOS doc

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description

https://jira.mesosphere.com/browse/DOCS-1949

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joel-hamill** [comment]
@meichstedt @jdef see https://github.com/dcos/dcos-docs/blob/bededd726fca72ed2d3d65a126742b1fc7d6a3df/1.9/deploying-services/marathon-parameters.md

File: 1.9/storage/persistent-volume.md [hunk 1]
```diff
@@ -73,10 +71,10 @@ The second volume is a persistent volume with a `containerPath` that matches the
 
 For a complete example, see the [Running stateful MySQL on Marathon](#stateful-sql) section.
 
-## Creating a stateful application via the DC/OS Web Interface
+## Creating a stateful application via the DC/OS GUI
```
Comment (meichstedt) [inline 1.9/storage/persistent-volume.md]: Is `DC/OS GUI` the official term? People in engineering usually refer to it as the `DC/OS UI` (leaving out the G(raphical)) but I can't tell how this is handled in DC/OS docs.

Comment (meichstedt) [inline 1.9/storage/persistent-volume.md]: Is `DC/OS GUI` the official term? People in engineering usually refer to it as the `DC/OS UI` (leaving out the G(raphical)) but I can't tell how this is handled in DC/OS docs.

File: 1.9/storage/persistent-volume.md [hunk 2]
```diff
@@ -142,7 +140,7 @@ However, if another framework does not respect the presence of labels and the se
 
 ### The Mesos Sandbox
 
-The temporary Mesos sandbox is still the target for the `stdout` and `stderr` logs. To view these logs, go to the Marathon pane of the DC/OS web interface.
+The temporary Mesos sandbox is still the target for the `stdout` and `stderr` logs. To view these logs, go to the Marathon pane of the DC/OS GUI.
```
Comment (meichstedt) [inline 1.9/storage/persistent-volume.md]: And finally `GUI` vs `UI` vs `web interface`.

To me personally, all are fine even in combination. Feel free to ignore these comments.

File: 1.9/storage/persistent-volume.md (hunk 1 above)
Comment (joel-hamill) [inline 1.9/storage/persistent-volume.md]: yes, GUI is the official term. there are occurrences of web interface and UI throughout - but our goal is to change to GUI :)

**Comment by joel-hamill** [comment]
@meichstedt re making changes to my branch - i'd prefer to make them myself for future references. it always creates some extra work in my branch after someone else commits to it (but not a huge deal). thanks for the feedback!

**Pull request merged**